File: PROYECTO_RESTFULL_JAVA_BDD/01 SERVIDOR/BanquitoCore/src/main/java/ec/edu/monster/model/Credito.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.monster.model;

/**
 *
 * @author victo
 */
import java.util.Date;


class Credito {
    private int idCredito;
    private int idCliente;
    private String cedula;
    private double montoCredito;
    private double tasaInteres;
    private int numeroCuotas;
    private double cuotaMensual;
    private String estado;
    private Date fechaOtorgamiento;
    
    // Constructor vacío
    public Credito() {}
    
    // Getters y Setters
    public int getIdCredito() { return idCredito; }
    public void setIdCredito(int idCredito) { this.idCredito = idCredito; }
    
    public int getIdCliente() { return idCliente; }
    public void setIdCliente(int idCliente) { this.idCliente = idCliente; }
    
    public String getCedula() { return cedula; }
    public void setCedula(String cedula) { this.cedula = cedula; }
    
    public double getMontoCredito() { return montoCredito; }
    public void setMontoCredito(double montoCredito) { this.montoCredito = montoCredito; }
    
    public double getTasaInteres() { return tasaInteres; }
    public void setTasaInteres(double tasaInteres) { this.tasaInteres = tasaInteres; }
    
    public int getNumeroCuotas() { return numeroCuotas; }
    public void setNumeroCuotas(int numeroCuotas) { this.numeroCuotas = numeroCuotas; }
    
    public double getCuotaMensual() { return cuotaMensual; }
    public void setCuotaMensual(double cuotaMensual) { this.cuotaMensual = cuotaMensual; }
    
    public String getEstado() { return estado; }
    public void setEstado(String estado) { this.estado = estado; }
    
    public Date getFechaOtorgamiento() { return fechaOtorgamiento; }
    public void setFechaOtorgamiento(Date fechaOtorgamiento) { this.fechaOtorgamiento = fechaOtorgamiento; }
}

File: EXAMEN_SOAP_DOTNET_BDD/01 SERVIDOR/01_SERVIDOR_BANCO/01_SERVIDOR_BANCO/BanQuito_Soap_Dotnet/ec.edu.espe.modelo/Credito.cs
using System;
using System.Runtime.Serialization;

namespace BanQuito_Soap_Dotnet.ec.edu.monster.modelo
{
    [DataContract(Name = "credito")]
    public class Credito
    {
        [DataMember(Order = 1)]
        public int IdCredito { get; set; }

        [DataMember(Order = 2)]
        public int IdCliente { get; set; }

        // ✅ NUEVO CAMPO
        [DataMember(Order = 3)]
        public string Cedula { get; set; }

        [DataMember(Order = 4)]
        public decimal MontoCreditoOtorgado { get; set; }

        [DataMember(Order = 5)]
        public decimal TasaInteres { get; set; }

        [DataMember(Order = 6)]
        public int NumeroCuotas { get; set; }

        // ✅ NUEVO CAMPO
        [DataMember(Order = 7)]
        public decimal CuotaMensual { get; set; }

        [DataMember(Order = 8)]
        public DateTime FechaOtorgamiento { get; set; }

        [DataMember(Order = 9)]
        public string Estado { get; set; }

        public Credito() { }

        public Credito(int idCredito, int idCliente, string cedula, decimal montoCreditoOtorgado,
                      decimal tasaInteres, int numeroCuotas, decimal cuotaMensual,
                      DateTime fechaOtorgamiento, string estado)
        {
            IdCredito = idCredito;
            IdCliente = idCliente;
            Cedula = cedula; //
            MontoCreditoOtorgado = montoCreditoOtorgado;
            TasaInteres = tasaInteres;
            NumeroCuotas = numeroCuotas;
            CuotaMensual = cuotaMensual; //
            FechaOtorgamiento = fechaOtorgamiento;
            Estado = estado;
        }
    }
}